module/applications/actor/base-sheet.mjs

```diff
@@ -816,14 +817,46 @@ export default class ActorSheet5e extends ActorSheet {
             keepMental: true,
             mergeSaves: true,
             mergeSkills: true,
-            transformTokens: rememberOptions(html).transformTokens
+            transformTokens: rememberOptions(html).transformTokens,
```

Comment:
I don't know if this will work, as it will take all the options selected in the dialogue, rather than using the defaults plus the over-rides that we want for these "quick select" options.

Reply:
You are right, I had forgotten that we don't take all the sheet options when clicking these buttons. In fact, it seems we're taking all the AE options from the sheet, but actually I think we should treat these the same as the rest of the options. For example, with Polymorph and Wildshape, you explicitly do not benefit from equipment, so AEs from equipment should always be disabled. For Polymorph, you shouldn't retain benefits from your class features, but in Wildshape you do, etc.

I also like @Varriount's suggestion to have these presets on the `CONFIG` object since they're getting a bit lengthy:

```js
CONFIG.transformationPresets = {
  wildshape: {
    icon: '<i class="fas fa-paw"></i>',
    label: "DND5E.PolymorphWildShape",
    options: {
      keepBio: true,
      removeEquipmentAE: true,
      // ...
    }
  }
};
```

Reply:
So the thing with wildshape is that sometimes you _can_ retain the item and effect, if the DM allows it/fits with the transformed creature, thus I left these as always customisable, as I suspect you can't please everyone with a default for the active effect removal/retention.

Reply:
The whole point of the presets is to provide a quick way to apply those specific transformation effects according to the rules. For polymorph, the rules are clear: You do not retain equipment benefits, you do not retain class/subclass feature benefits, so all of those active effects should be disabled by the preset and not left customisable.

For wildshape, you do always retain your class/subclass features so those active effects should always be enabled by the preset, etc. We do not offer a way to piecemeal select which equipment you do or do not retain the benefit of for wild shape, so keeping or removing active effects is going to be wrong in some case or another. In this case I would vote to set the preset to keep the active effects for equipment since that has been the status quo for this feature in our system until now.

Reply:
Okay, I'll make those changes, I'm away for the weekend so it will be next week.

Reply:
Okay, this is done.